sources/packages/backend/apps/queue-consumers/src/processors/schedulers/notification/models/notification.dto.ts

```diff
@@ -0,0 +1,9 @@
+export interface ProcessNotificationsQueueInDTO {
+  /** Maximum number of notifications to process per schedule */
+  pollingLimit: number;
```

Comment:
I was under the impression that this type of parameter (non-user inputted) would be acquired and used only by the consumer.
The scheduler would be sending a parameter to the queue in a few limited scenarios like IER12 when the default date would not be the one required. But this works too.